modules/internal/ChapelLocale.chpl

```diff
@@ -153,71 +162,150 @@ module ChapelLocale {
         this._instance = nil;
     }
 
+    pragma "no doc"
     proc init=(other: locale) {
       this._instance = other._instance;
     }
 
+    pragma "no doc"
     proc deinit() { }
 
+    /*
+      This is the maximum task concurrency that one can expect to
+      achieve on this locale.  The value is an estimate by the
+      runtime tasking layer.  Typically it is the number of physical
+      processor cores available to the program.  Creating more tasks
+      than this will probably increase walltime rather than decrease
+      it.
+     */
```

Comment:
This comment looks like a job for @gbtitus or @ronawho to proofread.

Reply:
Ah wait, I see now that it was quoted from pre-existing text, so likely they already have / wrote it.

Reply:
Yeah, I didn't really "document" anything myself in this PR, only the default locale value